### PR TITLE
Homework jdi

### PR DESCRIPTION
**Pull request by Kuzeikind**
Homework 7 done.

### CODE REVIEW

**Review by cold-flame — CHANGES_REQUESTED** [review]

File: src/main/java/com/epam/tc/hw7/JdiTestingSite.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import com.epam.tc.hw7.pages.HomePage;
+import com.epam.tc.hw7.pages.MetalsAndColorsPage;
+
+@JSite("https://jdi-testing.github.io/jdi-light/")
+public class JdiTestingSite {
+
+    public static HomePage homePage;
+    public static MetalsAndColorsPage metalsAndColorsPage;
+
+    @UI(HeaderLocators.HEADER_MENU_ITEM)
```
[the remark this answers is not in the document]
Comment (Kuzeikind) [reply src/main/java/com/epam/tc/hw7/JdiTestingSite.java]: Moved fields to HomePage.

File: src/main/java/com/epam/tc/hw7/locators/MetalsAndColorsPageLocators.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.epam.tc.hw7.locators;
+
+public class MetalsAndColorsPageLocators {
```
[the remark this answers is not in the document]
Comment (Kuzeikind) [reply src/main/java/com/epam/tc/hw7/locators/MetalsAndColorsPageLocators.java:3]: Deleted this class.

File: src/test/java/com/epam/tc/hw7/InitTest.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import com.epam.tc.hw7.enums.HeaderMenuItems;
+import com.epam.tc.hw7.pages.MetalsAndColorsPage;
+import com.epam.tc.hw7.util.Users;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+
+public interface InitTest {
+
+    @BeforeClass(alwaysRun = true)
```
[the remark this answers is not in the document]
Comment (Kuzeikind) [reply src/test/java/com/epam/tc/hw7/InitTest.java]: Deleted this class.

File: src/main/java/com/epam/tc/hw7/locators/HeaderLocators.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package com.epam.tc.hw7.locators;
+
+public class HeaderLocators {
```
[the remark this answers is not in the document]
Comment (Kuzeikind) [reply src/main/java/com/epam/tc/hw7/locators/HeaderLocators.java:3]: Deleted this class.

**Review by cold-flame — APPROVED** [review]

**Pull request merged**